app/labor/badge_rewarder.rb

```diff
@@ -17,11 +24,10 @@ def self.award_yearly_club_badges
     end
   end
 
-  def self.award_beloved_comment_badges
-    # ID 3 is the proper ID in prod. We should change in future to ENV var.
-    badge_id = Badge.find_by(slug: "beloved-comment")&.id || 3
-    Comment.where("public_reactions_count > ?", 24).find_each do |comment|
-      message = "You're famous! [This is the comment](https://#{ApplicationConfig['APP_DOMAIN']}#{comment.path}) for which you are being recognized. 😄"
+  def self.award_beloved_comment_badges(comment_count = 24)
+    badge_id = Badge.find_by!(slug: "beloved-comment").id
```

Comment:
I removed the hard-coded badge id and made this follow behave like `award_badges`. I used Blazer to confirm that a badge with the correct slug exists in production.

Reply:
@citizen428 whats the likelihood of other communities like mma not having this badge with the correct slug in production. Should we cater for that?

Reply:
Making badges completely optional was left for later. In the meantime I at least made the comment count configurable through an argument.